node/common/transport/amqp/lib/amqp.js

```diff
@@ -79,7 +80,7 @@ Amqp.prototype.connect = function connect(done) {
   }
 
   this._connectPromise
-    .then(function () {
+    .then(function (res) {
```

Comment:
Why add the parameter if you don't use it?  I'd assume that jshint would complain about this...